src/pyvesync/vesync.py

```diff
@@ -93,23 +115,66 @@ def get_energy_dict_from_api(self, response):
         return data
 
     def get_headers(self):
-        return {'Content-Type': 'application/json'}
+        header = {}
+        header['accept-language'] = 'en'
+        header['accountId'] = self.account_id
+        header['appVersion'] = APP_VERSION
+        header['content-type'] = 'application/json'
+        header['tk'] = self.tk
+        header['tz'] = self.time_zone
+        return header
+
+    def get_body(self, type):
```

Comment:
`type` is one of python's builtins keywords and generally should be avoided as a var name. using it here doesn't cause any issues but it might be preferable to use something like `type_` which is the pep8 recommendation

Reply:
Your right, I'll make the change